index.html

```diff
@@ -56,8 +64,8 @@ <h1>  I’m Lalla <br>
         <div class="grid-container">
             <section class = "cards">
                 <div class="job">
```

Comment:
great job in the section as well , also small adjustments to do :

1. the background color is a bit darker 
2. the horizontal margins of the cards needs to be reduced 
3. align the content of info section of the cards


![figma projects](https://user-images.githubusercontent.com/59707859/230373877-a724a463-08fd-4834-8492-c8c8f2038376.PNG)

![students projects](https://user-images.githubusercontent.com/59707859/230373896-42b87404-09f8-4ec3-b0ae-af593d955ac5.PNG)